### PR TITLE
Feature/SC-7447 - add warning text leaving sc

### PR DESCRIPTION
**Pull request by Ojself**
# Description
## Links to Tickets or other pull requests
Ticket: https://ticketsystem.hpi-schul-cloud.org/browse/SC-7447
Client: https://github.com/hpi-schul-cloud/schulcloud-client/compare/Feature/SC-7447-add-warning-text-leaving-sc?expand=1
Nuxt: https://github.com/hpi-schul-cloud/nuxt-client/compare/Feature/SC-7447-add-warning-text-leaving-sc?expand=1

The pr adds a warning text required by stake holders to warn users when they leave schul-cloud.
## Approval for review
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

client
nuxt

### CODE REVIEW

**Review by JenyMzo — COMMENTED** [review]
could you create classes to reuse the styles instead of using style attr? :)

File: src/components/organisms/LernstoreDetailView.vue [hunk 1]
```diff
@@ -59,6 +59,7 @@
 					<base-button
 						v-if="isMerlinContent"
 						design="outline"
+						style="width: 100%;"
```
Comment (virgilchiriac) [inline src/components/organisms/LernstoreDetailView.vue:62]: given this is only a temporary solution, it is ok to have inline style, but normally should be done with classes etc

**Review by virgilchiriac — APPROVED** [review]

File: src/components/organisms/LernstoreDetailView.vue [hunk 2]
```diff
@@ -72,11 +73,31 @@
...
+						}"
+						class="text-s"
+					>
+						{{ $t("pages.content.material.leavePageWarningMain") }}
+					</p>
+					<p
+						:style="{
+							color: 'var(--color-danger)',
+						}"
+						class="text-xs"
```
Comment (virgilchiriac) [inline src/components/organisms/LernstoreDetailView.vue]: this might only a temporary solution, but should be done with css classes etc

**Review by virgilchiriac — APPROVED** [review]

**Comment by JenyMzo** [comment]
@schul-cloud-bot update screenshots

**Review by virgilchiriac — APPROVED** [review]

**Pull request merged**